create-custom-map-style/tileserver/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "tileserver",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "aws4": "^1.11.0",
```

Comment:
I don't think `aws4` is needed.

Reply:
`aws4` I added to download the sprites/glyphs from Location Service and serve them from a local directory. What do you think?

Reply:
It looks like you're just using the AWS CLI to download these; can provided a link to the place where `aws4` is used?

Reply:
You're right, I did change how I was downloading them. I'm going to remove `aws4` too